packages/devtools_app/lib/src/screens/memory/panes/control/left_controls.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../../../analytics/analytics.dart' as ga;
+import '../../../../analytics/constants.dart' as analytics_constants;
+import '../../../../shared/common_widgets.dart';
+import '../../../../shared/theme.dart';
+import '../../memory_controller.dart';
+import 'constants.dart';
+import 'interval_dropdown.dart';
+
+class LeftControls extends StatefulWidget {
```

Comment:
rename left and right to primary and secondary to be consistent with other parts of DevTools